Forcefully disable the use of backtrace/backtrace_symbols which we don't
have in our libc.

Index: src/core/RDebug.cpp
--- src/core/RDebug.cpp.orig
+++ src/core/RDebug.cpp
@@ -37,7 +37,7 @@ QString RDebug::prefix;
 QMutex RDebug::mutexCounter;
 
 void RDebug::printBacktrace(const QString& prefix) {
-#if !defined(Q_OS_WIN) && !defined(Q_OS_ANDROID)
+#if !defined(Q_OS_WIN) && !defined(Q_OS_ANDROID) && !defined(Q_OS_OPENBSD)
     void *array[20];
     size_t size;
     char **strings;
